### DISCUSSION TITLE
如何想到獨一無二的創意點子？

### DISCUSSION THREAD

**Discussion by a3240281370** [1101-GS4719-C/Python-Community-GS4719-C#1405] [General]
最近看到新創千里馬和其他的創意競賽，很想參加，但想了幾個議題發現網路上早就一堆人做過了而且也都做得很完整，感覺現在要想出一個沒有人做過的議題相當困難，想請問有沒有什麼建議或好方法能幫助思考獨特的想法呢？

**Comment by TW-Maverick** [comment]
鄙人的建議是多多閱讀各種類的書籍或許能讓你激發不同的想法？其實不一定要有很創新的Idea，如果可以改進或優化前人的東西，未嘗不是一件好事。

**Comment by alicia0725** [comment]
我認為要有創意真的需要多出門、多看書，因為我自己以往參加這類比賽使用的方法就是在生活中找到不便的地方，然後嘗試想出能解決這些不便的方法，很有可能就是一個創新的點子了

**Comment by 1104030360** [comment]
我認為可以多跟朋友交流生活上的不便，從而獲取靈感，或是多閱讀一些書籍，從而得知問題點，也同時得到靈感

**Comment by Miinteaa** [comment]
我在參加其他的商業競賽時也有相同的困擾！後來經過一番探究發現除了閱讀的東西不夠多以外，思考也是一個很重要的環節！其實我們在日常生活中也能夠獲得很多資訊或者是啟發，只是沒有注意到而已。我的解決方法就是嘗試更仔細觀察身邊的人事物並把想到的任何連結或者疑惑紀錄下來！希望能夠幫助到你～

**Comment by aa103967622** [comment]
在參加這種創意競賽之前，我通常會閱讀相關雜誌文章，畢竟相關雜誌或網站上紀錄的就是最新的發展，有時候在閱讀這些新事物的同時，腦中會蹦出更加新穎的解決方法，就是大腦連結到這個領域的雲端上一樣，你可以在這些資料夾裡面自己挑出不同的東西，他們之間的化學反應可以激發你的靈感！

**Comment by alstinlin** [comment]
我覺得可以多跟不同系所的朋友交流，因為接觸的領域不同，對事情也會有不同見解，例如這堂課的小組討論就是不錯的時機，因為同組的同學幾乎來自不同科系，多多討論的話應該能想出不錯的點子

**Comment by ALAN30359704** [comment]
我認為要有獨一無二好的想法需要有紮實的基礎，並且能夠融會貫通的去聯想這些知識彼此的關聯性再加以結合，我認為如何把知識串連起來才是靈感的本質。

**Comment by ashleyy-huang** [comment]
萬事真的起頭難，我覺得創意的點子不一定要憑著自己來想出來，多多和身邊的人交流，或是上網尋找相關的資料一定可以激發出一些不錯的點子，有時候也可以把別人已經想出來的點子往反方向想，或許能夠有不一樣的收穫。

**Comment by Brownnnn78** [comment]
要找到新的而且沒有人想到的東西真的非常困難，總是有種要是我能發現什麼新的點子或許我就是天才，而不是每天為了考試存在的學生了，理想一直都很遠大，覺得自己總有一天會有什麼新點子或是變成一個非常獨特才能的人，但實際上卻一直是個分母，資訊的發達讓我們不會去創新而是一直學習別人發現的東西，雖然學習很重要但是卻也越學越迷茫。

**Comment by juliaouo** [comment]
我本身是沒有什麼創意力的人，不管是想 project 題目或是一些創新的東西都會很痛苦，很羨慕腦子裡有一堆點子的人，不知道如何才能提升自己的創意力？可能跟我很少跟不同類型的人來往有關，感覺多跟他人交流可以擦出不錯的火花，但對社交障礙有點困難就是了XD

**Comment by JessyCheng2003** [comment]
我認為主要是多聽、多看，除了多閱讀以外，也可以多看網路上的文章和影片，此外，多和不同的人聊天和相處也會很有幫助，還可以多出去走走、觀察世界。

**Comment by lyien1021** [comment]
我覺得如果是針對這種專題競賽，真的可以跟教授多交流，因為教授們的見識及經驗一定比我們豐富很多，能想得到的議題創意程度上也高很多。

**Comment by Evening218** [comment]
我覺得前人的成果大部分都是為了解決遇到的問題,所以其實可以試著將日常生活中看到的問題或是不便都記錄下來,應該可以給自己提供一個思考的方向,我覺得有了問題再去想答案應該會比較容易

**Comment by PaulSu0905** [comment]
我覺得當許多主題已經在網上被探討過時，尋找獨特的想法可能是一種挑戰。為了產生獨特的想法，要觀察現有的差距，進行跨學科思考，傾聽未滿足的需求，並結合個人的熱情和專業知識。這些方法可以培養創新和獨特的觀點。

**Comment by larry40305** [comment]
最重要的是保持開放的心態和持續的努力。創造獨特的想法需要時間和持之以恆的探索。堅持不懈，相信自己的能力，你將能夠找到那個獨一無二的議題和創意解決方案。

**Comment by ThanatosJun** [comment]
創意的訓練可以也在平日練習，當我們看見一件事或物，以其為中心，去思考如何向外擴散，例如現在我在打討倫文章，這主題是「創意想法」，向外可能有與朋友交流、靈光一現、多看書閱讀、畫結構圖、找網路論壇、問ChatGPT等，那如果我選擇其中一種對我自身有甚麼影響，我又能從那相影響中找到哪些社會議題這樣，這是我訓練自己創意的方式，提供參考。
